index.js

```diff
@@ -4,39 +4,45 @@
 module.exports = {
   name: 'ember-welcome-page',
 
+  init: function() {
+    this._super.init.apply(this, arguments);
+    var config = this.project.config();
```

Comment:
My only real blocker is this part, we shouldn't pull things that are not needed at browser runtime from config/environment.js. I think the only change to this PR would be to move this check to included hook and use `app.options['ember-welcome-page']` which pulled from ember-cli-build.js.